python/chemiscope/input.py

```diff
@@ -627,30 +627,36 @@ def quick_settings(
             structure viewer (following the chemiscope settings schema).
     """
 
-    if map_settings is None:
-        map_settings = {}
-
-    if structure_settings is None:
-        structure_settings = {}
-
-    map_settings.update(
-        {
+    if (x + y + z + color + size + symbol) == "":
+        # if at least one of the properties is requested
+        computed_map_settings = {}
+    else:
+        computed_map_settings = {
             "x": {"property": x},
             "y": {"property": y},
             "z": {"property": z},
             "color": {"property": color},
             "size": {"property": size},
             "symbol": symbol,
         }
-    )
 
-    structure_settings.update(
+    computed_map_settings.update(
         {
-            "keepOrientation": trajectory,
-            "playbackDelay": 10 if trajectory else 700,
+            "traceLine": trajectory,
```

Comment:
Should we check if the rendering mode is per-structure before setting this? I might be weird for per-atom properties

Reply:
good point. IDK if it's actually feasible in this function ATM but I'll look into it.